### PR TITLE
Task/add stream function

### PR DESCRIPTION
**Pull request by ferpessina**

### CODE REVIEW

File: job.js [hunk 1]
```diff
@@ -257,6 +291,15 @@ class BigQueryJob {
     _getQueryResults(job) {
         return job.getQueryResults({ autoPaginate: true });
     }
+
+    /**
+     * @param job
+     * @returns {stream.Readable}
```
Comment (maxiruani) [inline job.js:297]: I think this should be `@returns {Promise.<stream.Readable>}` instead

File: job.js [hunk 2]
```diff
@@ -249,6 +249,40 @@ class BigQueryJob {
...
+        try {
+            const [job] = await this.bigQuery.createQueryJob(jobOpts);
+            this.logger.info('bigquery-job.js Executed job. Getting query results stream. name:%s', this.name);
+
+            let stream;
+
+            let destinationMsg = this._getDestinationLogMsg();
+            let metadata;
+
+            stream = await this._getQueryResultsStream(job);
```
Comment (maxiruani) [inline job.js:271]: Use `const stream`

File: job.js [hunk 3]
```diff
@@ -249,6 +249,40 @@ class BigQueryJob {
...
+            this.logger.info('bigquery-job.js Got query results stream. name:%s elapsed:%s ms%s', this.name, elapsed.end(), destinationMsg);
+            [metadata] = await job.getMetadata();
```
Comment (maxiruani) [inline job.js:273]: Use `const [metadata]`

File: job.js [hunk 4]
```diff
@@ -249,6 +249,40 @@ class BigQueryJob {
...
+        this.logger.info('bigquery-job.js Running job. name:%s', this.name);
+        const jobOpts = this.getQueryOptions({ dryRun: false });
+
+        try {
+            const [job] = await this.bigQuery.createQueryJob(jobOpts);
+            this.logger.info('bigquery-job.js Executed job. Getting query results stream. name:%s', this.name);
+
+            let stream;
+
+            let destinationMsg = this._getDestinationLogMsg();
```
Comment (maxiruani) [inline job.js:268]: Use `const destinationMsg`

**Review by rodricad — APPROVED** [review]

**Pull request merged**